prometheus-exporters-formula/prometheus-exporters/init.sls

```diff
@@ -1,12 +1,45 @@
 {% from "prometheus-exporters/map.jinja" import exporters with context %}
 
+{% set proxy_enabled = salt['pillar.get']('proxy_enabled') %}
+
+exporter_exporter:
+{% if proxy_enabled %}
+  pkg.installed:
+    - name: {{ exporters.exporter_exporter_package }}
+  file.managed:
+    - name: {{ exporters.exporter_exporter_service_config }}
+    - source: {{ 'salt://prometheus-exporters/files/exporter-exporter-config.'
+        ~ salt['grains.get']('os_family') }}
+    - makedirs: True
+    - template: jinja
+    - user: root
+    - group: root
+    - mode: 644
+    - require:
+      - pkg: exporter_exporter
+    - watch_in:
+      - service: exporter_exporter
+  service.running:
+    - name: {{ exporters.exporter_exporter_service }}
+    - enable: True
+    - require:
+      - file: exporter_exporter
+{% else %}
+  service.dead:
+    - name: {{ exporters.exporter_exporter_service }}
+    - enable: False
+{% endif %}
+
+{% set node_exporter_enabled =
+  salt['pillar.get']('node_exporter:enabled', True) %}
 node_exporter:
-{% if salt['pillar.get']('node_exporter:enabled', True) %}
+{% if node_exporter_enabled %}
   pkg.installed:
     - name: {{ exporters.node_exporter_package }}
   file.managed:
     - name: {{ exporters.node_exporter_service_config }}
-    - source: {{ 'salt://prometheus-exporters/files/node-exporter-config.' ~ salt['grains.get']('os_family') }}
+    - source: {{ 'salt://prometheus-exporters/files/node-exporter-config.' ~
+        salt['grains.get']('os_family') }}
```

Comment:
I guess this is just a formatting change to avoid long lines, what is the maximum line length to use? Is there a specific styleguide for `sls` files you are following?

Reply:
I was just curious and don't have a strong preference. Just note that there are more occurrences to be reverted in case you do so, I just commented on this one as an example.

Reply:
Already done.